### PR TITLE
Fix key name

### PR DESCRIPTION
**Pull request by eldadfux**
Changed generic key name to `key` instead of `text`

### CODE REVIEW

File: tests/Database/Validator/QueriesTest.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ class QueriesTest extends TestCase
         'indexesInQueue' => [
             [
                 '$id' => 'testindex4',
-                'type' => 'text',
+                'type' => 'tekeyxt',
```
Comment (kodumbeats) [inline tests/Database/Validator/QueriesTest.php:122]: Is `tekeyxt` intentional for testing?

**Review by kodumbeats — CHANGES_REQUESTED** [review]
One question :)

File: tests/Database/Validator/QueriesTest.php (hunk 1 above)
Comment (eldadfux) [reply tests/Database/Validator/QueriesTest.php:122]: No, and yet the tests are green 😬

Comment (eldadfux) [reply tests/Database/Validator/QueriesTest.php]: fixed

**Pull request merged**